apps/artist/routes.coffee

```diff
@@ -18,6 +18,7 @@ sd = require('sharify').data
 
 @index = (req, res, next) ->
   tab = if req.params.tab? then req.params.tab else ''
+  showArtistCTA = req.query.show_artist_cta_code is SHOW_ARTIST_CTA_CODE
```

Comment:
https://github.com/artsy/force/blob/893279cb062508d4bb47f597d17997f7bfab7d1d/config.coffee#L131-L133

Keeping it as null (or some friendly default) would be fine, but we def wouldn't want the real value in here (esp. since this repo is now open!).